bin/sozo/src/commands/migrate.rs

```diff
@@ -80,71 +65,33 @@ impl MigrateArgs {
             return Err(anyhow!("Build project using `sozo build` first"));
         }
 
+        let MigrateArgs { name, world, starknet, account, .. } = self;
+
+        let name = name.unwrap_or_else(|| ws.root_package().unwrap().id.name.to_string());
```

Comment:
```suggestion
        let name = name.unwrap_or_else(|| ws.root_package().expect("Root package to be present.").id.name.to_string());
```
Or we may return an error instead panicking. But this is expected to be here anyway.